server/spec/test_helper.rb

```diff
@@ -6,4 +6,40 @@
   add_filter do |source_file|
     false
   end
+
+  class TestHelper < Object
+
+    ## return a fully qualified path to the test file directory
+    ## It assumes standard locations and looking for directory with name 'test-files'
+    def self.findTestFileDirectory
+      currentDirectory = Dir.pwd()
+
+      checkDir = "#{currentDirectory}/server/test-files"
+      return checkDir if Dir.exist?(checkDir);
+
+      checkDir = File.dirname(currentDirectory)+"/test-files"
+      return checkDir if Dir.exist?(checkDir);
+
+      puts "CAN NOT FILE TEST FILE DIRECTORY: ";
```

Comment:
Since this is only for testing I think it will be easy enough to figure out the directory by looking at the code.  The comment is very appropriate for production code.